### PR TITLE
Kubernetes Project Updates

### PR DESCRIPTION
**Pull request by vineeth-bandi**
*Issue #, if available:*

*Description of changes:*
Kubernetes project updates for 1.27 - 1.30

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/kubernetes/kubernetes/1-30/patches/0002-EKS-PATCH-Update-log-verbosity-for-node-health-and-t.patch [hunk 1]
```diff
@@ -5,6 +5,7 @@ Subject: [PATCH] --EKS-PATCH-- Update log verbosity for node health and taint
  checks
 
 cherrypick of https://github.com/kubernetes/kubernetes/pull/121967
+For more details https://t.corp.amazon.com/V1088905731
```
Comment (rcrozean) [inline projects/kubernetes/kubernetes/1-30/patches/0002-EKS-PATCH-Update-log-verbosity-for-node-health-and-t.patch:8]: We should remove this link

Comment (vineeth-bandi) [reply projects/kubernetes/kubernetes/1-30/patches/0002-EKS-PATCH-Update-log-verbosity-for-node-health-and-t.patch:8]: Yeah that makes sense. I'll make that change for all of those updated patches.

**Pull request merged**